components/concept/engine/src/main/java/mozilla/components/concept/engine/webextension/WebExtensionsTabsDelegate.kt

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.engine.webextension
+
+import mozilla.components.concept.engine.EngineSession
+
+/**
+ * Provides delegation for when a extension opens or closes a tab.
```

Comment:
This is a bit mechanic to read, plus we're not supporting "closing a tab". Let's file a follow-up for the latter and add a link.

I'd change the KDocs to something like: "Notifies applications / other components that a web extension wants to open a new tab via browser.tabs.create. Note that browser.tabs.remove is currently not supported: link to issue...